### ISSUE TITLE
[Deque Analysis] MyDSpace page's "serious" accessibility issues

### ISSUE THREAD

**Issue by tdonohue** [DSpace/dspace-angular#1186]
> Serious: This issue results in serious barriers for people with disabilities, and will partially prevent them from accessing fundamental features or content. People relying on assistive technologies will experience significant frustration as a result. Issues falling under this category are major problems, and remediation should be a priority. 

## Deque Analysis Summary

Our MyDSpace page as a total of 5 serious accessibility issues (not including color contrast which are handled in other tickets).

1. (ID 470746) "Hover content disappearing." on the tooltip/hints that appear when you hover over the action buttons (e.g. hover over "View" shows "Select this option to view the Item's metadata").  They all appear to be related to `ngb-tooltip-window`)
    * RULE: When additional content is triggered on pointer hover or on keyboard focus, that additional content MUST be visually perceived and MUST be dismissible.
    * HOW TO FIX: Fix this issue by ensuring that content that appears on mouse hover or keyboard focus meets ALL of the following requirements:
        * _Dismissible:_ If the content obscures or replaces other content, it can be dismissed via keyboard without moving mouse hover or keyboard focus - such as by pressing the Escape key.
        * _Hoverable:_ If the content is triggered by mouse hover, the user can move the mouse pointer to the content without the content disappearing.
        * _Persistent:_ The content stays visible until mouse hover or keyboard focus is moved or it is dismissed by the user (see #1 above), or the information is no longer valid.
        * See also https://www.w3.org/WAI/WCAG21/Understanding/content-on-hover-or-focus
2. (ID 470748) "Non-decorative content is inserted using CSS pseudo-elements." on the two buttons ("Add" and "Import" icons) which appear to the right of the "Drag & Drop your files" area.
    * RULE : Non-decorative content MUST NOT be inserted using CSS :before and :after pseudo-elements unless there is a way to access the content with CSS turned off.
    * HOW TO FIX: Fix this issue by including the content directly in the DOM using HTML.
3. ~~(ID 470750) "Buttons have same name but different actions."  on the "Edit" and "Delete" buttons.  Neither of them have a label/name, so they appear the same.~~
    * ~~Fix by adding an `aria-label` to both, or changing the button label (visible & programmatic) to provide a complete description of the button purpose.  See also https://dequeuniversity.com/class/forms2/labels/meaningful~~
    * FIXED BY #2092
4. (ID 470756, 470758) "Content is lost at 200% zoom" and "Content is lost at 320px width" occurs on the hover over text / tooltip (when hovering over a button).  In both cases the tooltip popup is partially hidden by the Admin sidebar menu. (screenshots available in the Deque tickets)

Full list of issues is viewable at (requires login): https://axeauditor.dequecloud.com/test-run/0856438a-a19a-11eb-bc31-b7d5be387c86/issues?activeTab=dt-issue&page=0&pageSize=50&sortField=ordinal&sortDir=asc&filter%5Bseverity%5D=3&filter%5Btype%5D=issue&filter%5Bpage_number%5D=10&row=9

## More Information / Tools

* Consider using [Deque's free Chrome plugin](https://chrome.google.com/webstore/detail/axe-devtools-web-accessib/lhdoppojpmngadmnindnejefpokejbdd?hl=en-US) to check your work
* Deque's color contrast tool: https://dequeuniversity.com/rules/axe/4.1/color-contrast?application=axeAPI

**Comment by tdonohue** [comment]
Reopening, as only part of the issues in the description have been resolved

**Comment by tdonohue** [comment]
Reanalyzed this ticket today after testing & merging #2683.  That PR appears to have fixed the final issues in this ticket. Therefore closing this as fixed in 7.6.2
